src/NzbDrone.Core/Housekeeping/HousekeepingService.cs

```diff
@@ -40,7 +41,7 @@ private void Clean()
                 }
             }
 
-            // Vacuuming the log db isn't needed since that's done hourly at the TrimLogCommand.
+            // Vacuuming the log db isn't needed since that's done in a separate housekeeping task
```

Comment:
Sure it does, its being called for the Log DB via a separate house keeping task, so we don't need to call it again here.